### PR TITLE
Added my profile details

### PR DESCRIPTION
**Pull request by Gautam-Hegde**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1083"></a>

### CODE REVIEW

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Update file name to GautamHedge.json instead of Gautam-Hedge.json, according to me this looks better.

**Comment by KendallDoesCoding** [comment]
Thanks

File: public/data/GautamHegde.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "milestones": [
+      {
+        "title": "Started Web Development",
+        "date": "June 2019",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started Learning Web Development"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/GautamHegde.json:27]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/GautamHegde.json (hunk 1 above)
Comment (KendallDoesCoding) [reply public/data/GautamHegde.json]: @Gautam-Hegde Thanks for adding this, please mark this as resolved.

**Review by Panquesito7 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]

**Comment by eddiejaoude** [comment]
> Update file name to GautamHedge.json instead of Gautam-Hedge.json, according to me this looks better.

@KendallDoesCoding their GitHub username is `Gautam-Hedge` so I think what they had first is better, I will update the docs to make it clearer for people to use their GitHub username (but at the end of the day it is their preference, it is their personal url)

**Comment by KendallDoesCoding** [comment]
> > Update file name to GautamHedge.json instead of Gautam-Hedge.json, according to me this looks better.
> 
> @KendallDoesCoding their GitHub username is `Gautam-Hedge` so I think what they had first is better, I will update the docs to make it clearer for people to use their GitHub username (but at the end of the day it is their preference, it is their personal url)

Yeah sorry about that, I only knew yesterday that it needs to be your github username, but according to me its better to have a url like linkfree.eddithub.org/KendallPinto instead of linkfree.eddithub.org/Kendall-Pinto, for eg.
'

**Pull request merged**